ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -287,7 +287,11 @@ open class SearchBar: UIView {
         if hidesNavigationBarDuringSearch {
             originalIsNavigationBarHidden = navigationController?.isNavigationBarHidden ?? false
 
```

Comment:
so why did the delayasync get moved?

Reply:
The delay async is only needed when the user cancels/exits search using the UI. It is not needed when setting hidesNavigationBarDuringSearch since the hide navigation bar isn't animated.